### ISSUE TITLE
statut stay pending

### ISSUE THREAD

**Issue by jssolutec** [naparuba/shinken#1885]
Hello,

I've just configured a new Shinken install, and active check won't works : 

![bug_shinken](https://cloud.githubusercontent.com/assets/24813112/21526513/1d3dce7e-cd25-11e6-8b09-64fe7f4214e4.JPG)

as you can see, the host stay for ever "pending",  and next active check is ago.

I really don't know how to solve. I'm on Debian 8, with Shinken 2.4.3

Thanks,

Js.

**Comment by geektophe** [comment]
Could you give more context about your configuration ?

- Which host template do you use (standard or custom)
- Are active checks enabled ?
- Which check_period do you use ?
- Does your poller(s) correctly communicate with your scheduler ?
- Is it the only host/service affected, or are there more ? (all ?)

**Comment by jssolutec** [comment]
Of course !

I used the standard localhost template, but I also have tried with custom one, same results.

Active checks are enabled.

I used 24x7 check perdiod, without modification.

I guess my poller and scheduler communicate correctly, I use default configuration, and arbiter's check say that everything looks ok.

All my hosts/services are affected.

Now Shinken greatly running on my new system : debian 7.2. On the same hardware, I just change the OS.

Really don't know what happen :/

**Comment by ThomasBarach** [comment]
Hi,

I have got the same error from a fresh install 2.4.3.

I tried to add the check_command directly on the host.cfg but does not work either.

Thanks

**Comment by bsautel** [comment]
Hi @jssolutec and @ThomasBarach.

I have a similar issue, all my hosts and services are down. with shinken 2.4.3.

I have many network errors in the broker log files. I reported them in issue #1895.

I am on Ubuntu Xenial and not Debian 8 but we are maybe facing the same issue.

Do you have the same error messages in your broker log file?

**Comment by ThomasBarach** [comment]
Hello @bsautel,

I'm also on Ubuntu Xenial. Yes, I have the same error message about the poller. 

I got it working by restarting the poller by hand but this is not a good solution. I tried to find the problem but whitout success.

**Comment by bsautel** [comment]
Thanks @ThomasBarach for your answer.

After restarting the poller by hand, is it working all the time? After multiple shinken restarts, I could get it initialize hosts and service states, but they could not be updated.

And how do you restart only one shinken process by hand? I should give it a try.

**Comment by ThomasBarach** [comment]
I restarted the poller once, then I restart only the arbiter if I need to deploy new configuration.

You can do `/etc/init.d/shinken-process restart` to restart one process.

Hope it helps

**Comment by bsautel** [comment]
I tried to restart several components and now it works.

I wish it is going to be stable as of now.

Thanks @ThomasBarach!

**Comment by ThomasBarach** [comment]
You're welcome @bsautel but as I said this is not a good solution. 
I do not use this Shinken in production I deployed it to test some things. 

Glad I could help you!

**Comment by bsautel** [comment]
Yes, you are right, it is not a satisfying solution. It unlocked checks just after restart but it does not last.

Since shinken does not work for me and seems to be no longer maintained, I am gonna have to switch to another monitoring tool. 😞

**Comment by small1** [comment]
Is anyone working on this at all?
